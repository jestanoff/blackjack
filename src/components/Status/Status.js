import React from 'react'
import { Text } from '../'
import PropTypes from 'prop-types'
import cx from 'classnames'
import './status.css'

const Status = ({ sumPlayer, sumAi }) => (
  <Text>{sumPlayer > sumAi && sumPlayer <= 21 ? 'Player Wins' : 'Dealer Wins'}</Text>
)

Status.propTypes = {
  sumPlayer: PropTypes.number.isRequired,
  sumAi: PropTypes.number.isRequired,
}

export default Status
