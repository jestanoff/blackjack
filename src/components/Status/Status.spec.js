import React from 'react'
import Status from './Status'
import renderer from 'react-test-renderer'

describe('<Status />', () => {
  it('should render proper markup', (): void => {
    const props = {
      sumAi: 1,
      sumPlayer: 22,
    }
    const tree = renderer.create(<Status {...props} />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
