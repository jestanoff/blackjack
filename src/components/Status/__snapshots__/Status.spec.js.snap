// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Status /> should render proper markup 1`] = `
<h1
  className="text text-color-dark-grey text-size-standard"
>
  Dealer Wins
</h1>
`;
